packages/components/src/local/molecules/json-editor/index.tsx

```diff
@@ -230,12 +235,14 @@ export const JsonEditor: React.FC<Props> = ({
   }, [disableArrayToolsWithEditor && disabled])
 
   useLayoutEffect(() => {
```

Comment:
@lilitkarapetyan - I tried to understand what was happening in this method before, and couldn't get it.

My favourite outcome would be to simplify the API/props for the `json-editor`.  I feel we've got some props that do the same job.  If we can't do that, I'll need better comments on the props, and in the method please.

Reply:
I will look into it.

Reply:
I don't remember where we use it either, I guess we can remove it from the `jsonEditor`